### PR TITLE
e2e_scheduling:stop using deprecated framework.ExpectError

### PR DESCRIPTION
**Pull request by Rei1010**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
test/e2e/scheduling stop using deprecated framework.ExpectError

#### Which issue(s) this PR fixes:
Fixes # https://github.com/kubernetes/kubernetes/pull/115961

#### Special notes for your reviewer:
NONE

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

File: test/e2e/scheduling/limit_range.go [hunk 1]
```diff
@@ -37,10 +37,11 @@ import (
 	"k8s.io/apimachinery/pkg/watch"
 	"k8s.io/client-go/tools/cache"
 	watchtools "k8s.io/client-go/tools/watch"
+	admissionapi "k8s.io/pod-security-admission/api"
```
Comment (wackxu) [inline test/e2e/scheduling/limit_range.go:40]: why update the import order? remove the empty line and keep the original code

File: test/e2e/scheduling/preemption.go [hunk 2]
```diff
@@ -39,12 +39,13 @@ import (
...
 	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/cache"
+	admissionapi "k8s.io/pod-security-admission/api"
+
 	"k8s.io/kubernetes/pkg/apis/scheduling"
 	"k8s.io/kubernetes/test/e2e/framework"
 	e2enode "k8s.io/kubernetes/test/e2e/framework/node"
 	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
 	e2ereplicaset "k8s.io/kubernetes/test/e2e/framework/replicaset"
-	admissionapi "k8s.io/pod-security-admission/api"
```
Comment (wackxu) [inline test/e2e/scheduling/preemption.go:47]: ditto

**Comment by Rei1010** [comment]
@wackxu @chendave @damemi ptal, thx

**Comment by Rei1010** [comment]
@kerthcet ptal

**Review by kerthcet — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**